website/addons/s3/tests/test_serializer.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+"""Serializer tests for the S3 addon."""
+import mock
+from nose.tools import *  # noqa (PEP8 asserts)
+
+from website.util import web_url_for
+from website.addons.base.testing.serializers import StorageAddonSerializerTestSuiteMixin
+from website.addons.s3.tests.factories import S3AccountFactory
+from website.addons.s3.serializer import S3Serializer
+
+from tests.base import OsfTestCase
+
+
+class TestS3Serializer(StorageAddonSerializerTestSuiteMixin, OsfTestCase):
+    required_settings = ('userIsOwner', 'nodeHasAuth', 'urls', 'userHasAuth',)
+    required_settings_authorized = ('ownerName', )
```

Comment:
Do `required_settings` and `required_settings_authorized` need to be defined here? They are already defined in `StorageAddonSerializerTestSuiteMixin`